src/components/shared/ProjectRiskNotice.tsx

```diff
@@ -38,8 +38,8 @@ export default function ProjectRiskNotice({
         </Trans>
       </p>
       <ul>
-        {warnings.map(text => (
-          <li>{text}</li>
+        {warnings.map((text, i) => (
+          <li key={`warning_${i}`}>{text}</li>
```

Comment:
wont comment on more of these but unsure if extra text buys us anything

Reply:
Yeah they def don't, thanks for calling out. My original understanding was that keys should be "globally" unique, but that's not correct 🙌 . Changed!